index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+   <main>
+        <nav>
+            <ul>
+                <li><a href="#"><div></div></a></li>
+                <span></span>
+                <li><a href="#">Store</a></li>
+                <span></span>
+                <li><a href="#">Mac</a></li>
+                <span></span>
+                <li><a href="#">iPod</a></li>
```

Comment:
What are these `span` for? If it is for the line separator between the links, you can add its styles inside the `<li>` styles itself as `border-right` without the need to repeat yourself.
Also if you want to exclude the last element (search item) from adding `border-right` you can use [`not(:last-child)`](https://stackoverflow.com/questions/7272302/excluding-last-child?noredirect=1&lq=1) css selector